data/examples/Computed+Provision-VU#290915_Coordinator-v2.01.json

```diff
@@ -0,0 +1,396 @@
+{
+  "timestamp": "2020-12-10T18:58:04.153Z",
+  "role": "Coordinator",
+  "id": "VU#290915",
+  "version": "2.0",
+  "computed": "SSVCv2/E:P/V:R/T:P/M:H/D:A/1607626684/",
```

Comment:
I'd still like to see version be separate from "SSVC", which I take as a namespace rather than a version, but that's also covered in #110 so I'm ok with making that be a separate task if the decision is to defer it here.

Reply:
Similarly, indication of which tree the vector addresses, also in #110. A separate pull request later for that is fine too.

Reply:
I think SSVC/2/... makes sense, that is, i'm in favor of separating the name space and the version.